### DISCUSSION TITLE
Garbo is not working after using svn update

### DISCUSSION THREAD

**Discussion by CwisGunza** [loathers/garbage-collector#1942] [General]
Hello everyone, I keep getting this message after I updated Garbo with svn update"
Using adventure value 4750.
JavaScript exception: KolmafiaVersionError: 'file:/C:/Users/Admin/OneDrive/Desktop/Mafia/scripts/garbage-collector/garbo.js' requires revision r27974 of kolmafia or higher (current: 27969). Up-to-date builds can be found at https://ci.kolmafia.us/. (file:/C:/Users/Admin/OneDrive/Desktop/Mafia/scripts/garbage-collector/garbo.js#21460)
at file:/C:/Users/Admin/OneDrive/Desktop/Mafia/scripts/garbage-collector/garbo.js:21460 (sinceKolmafiaRevision)
at file:/C:/Users/Admin/OneDrive/Desktop/Mafia/scripts/garbage-collector/garbo.js:49927 (main)

What should I do?

**Comment by ajcoppa** [comment]
The error message is telling you what you need to do – download a newer version of mafia with the link provided.

**Reply by CwisGunza** [reply]
I'm so sorry

**Comment by CwisGunza** [comment]
How do I delete this thread? I'm embarassed. I'm so sorry

**Reply by ajcoppa** [reply]
No need to delete, it's all good! Just explaining so that next time you see the error message you know what to do. And if you leave it up, others may search and see it too.
